### PR TITLE
Small random improvements to the step-by-step Manager guides

### PR DESCRIPTION
**Pull request by KobeW50**
These changes do not resolve the issues that I recently opened

### CODE REVIEW

File: step-by-step-general/01-requirements.md [hunk 1]
```diff
@@ -7,10 +7,8 @@ You need to have the following to follow this guide:
     - Android 8 or later *
     - ≥1GB storage space
 
-- About 15 minutes of time (depends on your device)
-    - speedrun current record: 4:15
+- About 10 minutes of time (depends on your device)
     - flagship: estimated 5 ~ 10 min
-    - mid-tier: estimated 10 ~ 15 min
-    - low-tier: estimated 15 ~ 35 min
+    - mid to low tier: estimated 10 ~ 15 min
```
Comment (KobeW50) [inline step-by-step-general/01-requirements.md:12]: I made these changes bec the current times seemed to reflect the old patching times, which have since drastically improved

File: step-by-step-general/04-check-compat.md [hunk 2]
```diff
@@ -2,20 +2,16 @@
...
 2. Go to the settings tab and scroll down until you find "About"
 ==- Image for reference
 ![check about](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/050-check_about.jpg?raw=true)
 ===
-3. Check that `Arch` had at least 1 entry that looks like `arm64-v8a`
-4. If you only got something that said `armv7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-5. If you got `armv8`, you are probably fine
-6. Check that your Android version is at least 8
-7. If you only got something that said `armv7` or lower, you may not be able to patch on your phone. You can patch it on another device or a computer with [ReVanced CLI](/06-revanced-cli.md).
-8. If you made it here, you can probably patch on your phone with ReVanced Manager
```
Comment (KobeW50) [inline step-by-step-general/04-check-compat.md:18]: I'm pretty sure the Manager won't be able to be installed at all on android 7 and below. I asked kitadai on discord to verify. Waiting to hear back

If this is true, a troubleshooting tip must be added to step 3 for users who are unable to install the manager

File: step-by-step/04-check-compat.md [hunk 3]
```diff
@@ -1,24 +1,20 @@
...
 2. Go to the settings tab and scroll down until you find "About"
 ==- Image for reference
 ![check about](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/050-check_about.jpg?raw=true)
 ===
-3. Check that `Arch` had at least 1 entry that looks like `arm64-v8a`
-4. If you only got something that said `armv7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-5. If you got `armv8` or higher, you are probably fine
-6. Check that your Android version is at least 8
-7. If you only got something that said `7` or lower, you cannot patch it on your phone. Please patch it on another device or your PC.
-8. If you made it here, you can probably patch on your phone with ReVanced manager
```
Comment (KobeW50) [inline step-by-step/04-check-compat.md:18]: I'm pretty sure the Manager won't be able to be installed at all on android 7 and below. I asked kitadai on discord to verify. Waiting to hear back

If this is true, a troubleshooting tip must be added to step 3 for users who are unable to install the manager

File: step-by-step-general/04-check-compat.md [hunk 4]
```diff
@@ -2,22 +2,18 @@
...
+3. Check that `Arch` says `arm64-v8a`
+4. If it does not say `arm64-v8a` you cannot patch YouTube on your device. Please patch it on another device or your PC.
 
-==- What does `arch` mean? Why is `armv7` not supported?
+==- What does `arch` mean? Why is `armeabi-v7a` not supported?
```
Comment (SodaWithoutSparkles) [inline step-by-step-general/04-check-compat.md:16]: `armeabi-v7a` means emulated armv7 (32 bit). This appears on devices that has `armv8` but wanted to retain compatibility with older apks. 

`armv7` is the intended.

File: step-by-step/04-check-compat.md [hunk 5]
```diff
@@ -1,24 +1,20 @@
...
+3. Check that `Arch` says `arm64-v8a`
+4. If it does not say `arm64-v8a` you cannot patch YouTube on your device. Please patch it on another device or your PC.
 
-==- What does `arch` mean? Why is `armv7` not supported?
+==- What does `arch` mean? Why is `armeabi-v7a` not supported?
```
Comment (SodaWithoutSparkles) [inline step-by-step/04-check-compat.md:16]: `armeabi-v7a` means emulated armv7 (32 bit). This appears on devices that has `armv8` but wanted to retain compatibility with older apks. 

`armv7` is the intended.

**Review by SodaWithoutSparkles — CHANGES_REQUESTED** [review]
`armv7` is actually the intended phrase

File: step-by-step-general/04-check-compat.md [hunk 6]
```diff
@@ -2,22 +2,18 @@
...
 `arch` is the architecture of your CPU. In layman terms, it is "the language of the CPU", if you don't know that language, you cannot communicate with that CPU. 
 
 The ReVanced team will move to `arsclib` soon™, which supports `armv7`. For now, you can patch on a computer with [ReVanced CLI](/06-revanced-cli.md) instead.
```
Comment (SodaWithoutSparkles) [inline step-by-step-general/04-check-compat.md:19]: `arsclib` seems to have no progress and ceased development. Would removing this line be approiate? Or should we hope that it revived some day?

File: step-by-step/04-check-compat.md [hunk 7]
```diff
@@ -1,24 +1,20 @@
...
 `arch` is the architecture of your CPU. In layman terms, it is "the language of the CPU", if you don't know that language, you cannot communicate with that CPU. 
 
-The ReVanced team will move to `arsclib` soon™, which supports `armv7`. For now, you can patch on a computer with `revanced-cli` or `revanced-builder` instead.
+The ReVanced team will move to `arsclib` soon™, which supports `armeabi-v7a`. For now, you can patch on a computer with `revanced-cli` or `revanced-builder` instead.
```
Comment (SodaWithoutSparkles) [inline step-by-step/04-check-compat.md:19]: `arsclib` seems to have no progress and ceased development. Would removing this line be approiate? Or should we hope that it revived some day?

**Review by SodaWithoutSparkles — COMMENTED** [review]
`arsclib` seems to have no progress and ceased development. Would removing this line be approiate? Or should we hope that it revived some day?

File: step-by-step-general/04-check-compat.md (hunk 4 above)
Comment (KobeW50) [reply step-by-step-general/04-check-compat.md:16]: I believe the Manager says "armeabi-v7a" on armv7 devices. The guide should refer to it by whatever the manager calls it, since users don't know that armv8 = arm64-v8a  or that armv7 = armeabi-v7a.

This screenshot is old, but I'm pretty sure it still says this and not "armv7"


If the manager did say "armv7" it would make sense for the guide to refer to it as such.

**Comment by SodaWithoutSparkles** [comment]
according to the team member named "arsclib", it is not being worked on

> arsclib branch is not currently being worked on

I think it should be safe to remove those lines

https://discord.com/channels/952946952348270622/952946952348270626/1259902169973526569

File: step-by-step/04-check-compat.md [hunk 8]
```diff
@@ -13,8 +13,6 @@ We are going to check if your device is supported.
 3. Check that `Arch` says `arm64-v8a`
 4. If it does not say `arm64-v8a` you cannot patch YouTube on your device. Please patch it on another device or your PC.
 
-==- What does `arch` mean? Why is `armeabi-v7a` not supported?
-`arch` is the architecture of your CPU. In layman terms, it is "the language of the CPU", if you don't know that language, you cannot communicate with that CPU. 
-
-The ReVanced team will move to `arsclib` soon™, which supports `armeabi-v7a`. For now, you can patch on a computer with `revanced-cli` or `revanced-builder` instead.
+==- What does `arch` mean? Why is `arm64-v8a` required?
+`arch` is the architecture of your CPU. In layman terms, it is "the language of the CPU". If you don't know that language, you cannot communicate with that CPU. If your device does not support `arm64-v8a` you can patch on a computer with the [`revanced-cli`](https://sodawithoutsparkles.github.io/revanced-troubleshooting-guide/06-revanced-cli/), [`Taku's auto-cli`](https://github.com/taku-nm/auto-cli) or [`revanced-builder`](https://github.com/reisxd/revanced-builder).
```
Comment (KobeW50) [inline step-by-step/04-check-compat.md:17]: Is the revanced-cli link correct? I wasn't sure if it should link to the `.md` file.

Comment (SodaWithoutSparkles) [reply step-by-step/04-check-compat.md:17]: Its fine, but using `/06-revanced-cli.md` would be better because it retains compatibility with github's view

**Review by SodaWithoutSparkles — APPROVED** [review]
LGTM

**Comment by SodaWithoutSparkles** [comment]
website is building, should be done in a minute or two
